code/modules/mob/living/carbon/human/species/xenomorphs/alien_species.dm

```diff
@@ -27,7 +41,7 @@
 	virus_immune = 1
 
 	brute_mod = 0.25 // Hardened carapace.
-	burn_mod = 2    // Weak to fire.
+	burn_mod = 1.25    // Weak to fire.
```

Comment:
They are quite difficult to kill already, so I am not sure about that.

They already have a 75% damage reduction from brute.
Reducing the laser damage to only 25% more seems a bit too little.

Either increase the brute mod to something like 0.75 or the burn mod.
Otherwise they are going to be extremely difficult to kill.
(Especially when hunting in packs, as they should)